.pre-commit-config.yaml

```diff
@@ -16,6 +16,15 @@ repos:
       - id: check-added-large-files
       - id: mixed-line-ending
 
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --remove-all-unused-imports
+          - --ignore-init-module-imports
```

Comment:
Move this up in the config. This way if any imports are dropped they are sorted by isort in the next step. Before this would take two runs to happen.